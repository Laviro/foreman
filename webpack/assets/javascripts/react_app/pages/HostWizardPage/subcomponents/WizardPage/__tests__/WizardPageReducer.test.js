import { testReducerSnapshotWithFixtures } from 'react-redux-test-utils';
import reducer from '../WizardPageReducer';

const fixtures = {
  'should return the initial state': {},
};

describe('WizardPage reducer', () =>
  testReducerSnapshotWithFixtures(reducer, fixtures));
