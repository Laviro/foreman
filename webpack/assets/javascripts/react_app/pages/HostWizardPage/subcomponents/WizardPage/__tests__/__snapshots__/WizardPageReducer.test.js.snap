// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WizardPage reducer should return the initial state 1`] = `
Object {
  "filter": "",
  "items": Array [],
  "itemsPage": 1,
  "selectedID": null,
}
`;
